<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartDesignGui::TaskLoftParameters</class>
 <widget class="QWidget" name="PartDesignGui::TaskLoftParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="PartDesignGui::FeaturePickerDoublePanelWidget" name="pickerWidget" native="true"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxRuled">
     <property name="text">
      <string>Ruled surface</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxClosed">
     <property name="text">
      <string>Closed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUpdateView">
     <property name="text">
      <string>Update view</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PartDesignGui::FeaturePickerDoublePanelWidget</class>
   <extends>QWidget</extends>
   <header>Mod/PartDesign/Gui/FeaturePickerWidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
